plugins/external-action/mocks/index.ts

```diff
@@ -0,0 +1,7 @@
+export interface Mock {
+  group: string;
+  name: string;
+  path: string;
+}
+
+export type Manifest = Array<Mock>;
```

Comment:
Is this file necessary?

Reply:
If so, could we put in a common place to reference from each individual mocks JS package?

Reply:
I'm having trouble accessing this file from other places with a single source.

I tried moving this to the root of plugins/index.ts

and within the BUILD file, I made this a filegroup

```
filegroup(
    name = "mock_types",
    srcs = glob(["*.ts"]),
    visibility = ["//visibility:public"],
)
``` 

within one of the plugins/mocks folder (in this case i used `plugins/pubsub/mocks/BUILD` 
I have a rule that copies this over to the root of the index folder 

```
genrule(
    name = "copy",
    srcs = ["//plugins:mock_types"],
    outs = ["index.ts"],
    cmd = "cp -r $(SRCS) $(OUTS)"
)

javascript_pipeline(
    name = "@player-ui/pubsub-plugin-mocks",
    entry = "index.ts",
    dependencies = [
        "@npm//@player-tools/dsl",
        ":copy"
    ],
    other_srcs = [
       "index.ts",
        ":manifest",
    ] + glob(["**/*.tsx"]),
    out_dir = "",
)
```



with the error being: 

ERROR: /Users/mercillo/Documents/intuit/playerOS/player/plugins/pubsub/mocks/BUILD:23:20: output 'plugins/pubsub/mocks/package.json' was not created
ERROR: /Users/mercillo/Documents/intuit/playerOS/player/plugins/pubsub/mocks/BUILD:23:20: Action plugins/pubsub/mocks/package.json failed: not all outputs were created or valid

would this be the right approach haha